pkg/controller/compliancescan/scantype.go

```diff
@@ -414,7 +414,29 @@ func (ph *platformScanTypeHandler) shouldLaunchAggregator() (bool, string, error
 func (ph *platformScanTypeHandler) gatherResults() (compv1alpha1.ComplianceScanStatusResult, bool, error) {
 	var result compv1alpha1.ComplianceScanStatusResult
 	isReady := true
+	// For CEL we don't need to check the ConfigMap
```

Comment:
This is because the CEL scanner container creates the CCR directly, right?

Reply:
that's correct, CEL has skipped any ConfigMaps steps compared to OpenSCAP scans